app/client/jsx/JitsiVideo.jsx

```diff
@@ -18,10 +18,17 @@ class JitsiVideo extends Component {
             'tileview', 'videobackgroundblur', 'download', 'help', 'mute-everyone',
             'e2ee'
         ]
+
         // These should not be stored in state for the duration of the Jitsi video,
         // since any change to state currently triggers a render and reconnect
         this.isAudioMuted = this.props.isAudioMuted
         this.isVideoMuted = this.props.isVideoMuted
+
+        // If this is a muteRoom, mute and remove microphone controls.
+        if (this.props.jitsiData.muteRoom) {
+            _.pull(this.toolbarButtons, 'microphone')
+            this.isAudioMuted = true;
+        }
```

Comment:
While I could've done `this.isAudioMuted = this.props.jitsiData.muteRoom || this.props.isAudioMuted`, I thought it would be better to put all the `muteRoom` options in one place.